src/node/coinstats.cpp

```diff
@@ -117,9 +138,17 @@ static void PrepareHash(CHashWriter& ss, const CCoinsStats& stats)
     ss << stats.hashBlock;
 }
 static void PrepareHash(std::nullptr_t, CCoinsStats& stats) {}
+// Muhash does not need the prepare step
```

Comment:
726c9504 pico-nit, feel free to ignore, s/Muhash/MuHash/

Reply:
Done